src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4441,7 +4441,7 @@ public override bool PreVisit(CallNode node)
                 Contracts.Assert(carg > 0);
 
                 // The zeroth arg should not be a lambda. Instead it defines the context type for the lambdas.
-                Contracts.Assert(!maybeFunc.IsLambdaParam(0));
+                Contracts.Assert(!maybeFunc.IsLambdaParam(null, 0));
```

Comment:
>null [](http://example.com/codeflow?start=58&length=4)

Why not pass the first argument? If we can avoid null values being passed to the virtual, then we should do that.